pom.xml

```diff
@@ -29,10 +29,9 @@
     <properties>
         <revision>3.10.0</revision>
         <changelist>-SNAPSHOT</changelist>
-        <gitHubRepo>jenkinsci/config-file-provider-plugin</gitHubRepo>
+        <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
         <maven.javadoc.skip>true</maven.javadoc.skip>
         <jenkins.version>2.303.3</jenkins.version>
-        <java.level>8</java.level>
```

Comment:
As of the update to parent POM 4.40, `java.level` is deprecated and inferred from the Jenkins core version. So this is no longer needed. I am deleting it to silence the deprecation warning.